### DISCUSSION TITLE
Lutris instead of/along ProtonDB

### DISCUSSION THREAD

**Discussion by danielfreire-dev** [Heroic-Games-Launcher/HeroicGamesLauncher#589] [Ideas]
Given the fact that there a lot of exclusives in the Epic Games Store, and more are expected to come, shouldn't [Lutris](https://lutris.net) be the go-to database to find out if you can run the game in Linux or not. It would also be the best place to share fixes. Or, at least, Heroic could provide both solutions.

**Comment by flavioislima** [comment]
This looks like a discussion and not a Heroic Issue.
I'll convert it to a discussion.
